views/components/BarGraph/index.jsx

```diff
@@ -30,52 +43,99 @@ const BarGraph = ({ graph }) => {
   }
 
   return (
-    <ResponsiveContainer width="100%" height={500}>
-      <BarChart width={600} height={400} data={sanitizeSiteData(siteData)}>
-        <Legend
-          verticalAlign="top"
-          height={50}
-          margin={{ top: 0, left: 0, right: 0, bottom: 20 }}
-        />
-        <XAxis
-          dataKey="name"
-          height={100}
-          angle={55}
-          textAnchor="start"
-          interval={0}
-        />
-        <YAxis width={80}>
-          <Label value="Percent" angle={-90} />
-        </YAxis>
-        <Tooltip
-          content={<BarGraphTooltip studyTotals={graph.studyTotals} />}
-        />
-        {graph.labels.map((label) => {
-          return (
-            <Bar
-              key={label.name}
-              dataKey={`percentages[${label.name}]`}
-              name={label.name}
-              id={label.name}
-              stackId="a"
-              maxBarSize={80}
-              fill={label.color}
-            >
-              <LabelList
-                valueAccessor={(entry) => {
-                  const percent = entry.percentages[label.name]
+    <Paper
+      sx={{
+        width: '100%',
+      }}
+    >
+      <ResponsiveContainer width="100%" height={500}>
+        <BarChart
+          id="recharts-wrapper"
+          data={sanitizeSiteData(siteData)}
+          barSize={50}
+          margin={{ top: 50 }}
+          onMouseMove={(data) => {
+            if (data.isTooltipActive) {
+              const chartWidth = Number(
+                document.getElementById('chartWidth').attributes.width.value
+              )
+              const xCoordinate = data.activeCoordinate.x
 
-                  if (percent && percent > 0) {
-                    return `${percent.toFixed(0)}%`
-                  }
-                }}
-                fill="white"
+              handleTooltipPosition({ chartWidth, xCoordinate })
+            }
+          }}
+          barCategoryGap={40}
+        >
+          <Legend
+            margin={{ bottom: 10 }}
+            verticalAlign="bottom"
+            align="left"
+            wrapperStyle={{
+              paddingLeft: '80px',
+              bottom: 50,
+            }}
+          />
+          {areMoreThanTenSites ? (
```

Comment:
Same question here, what makes 10 sites the magic number?

Reply:
In the design the number is 6 and I saw we can fit 10 comfortably on the screen. 

<img width="941" alt="Screenshot 2024-03-14 at 8 10 30 AM" src="https://github.com/AMP-SCZ/dpdash/assets/19805355/48c2a268-448c-42ff-b6df-8037a3e693a0">